models/src/main/scala/it/pagopa/interop/agreementmanagement/model/persistence/events.scala

```diff
@@ -12,3 +12,8 @@ final case class AgreementConsumerDocumentAdded(agreementId: String, document: P
     extends Event
 final case class AgreementConsumerDocumentRemoved(agreementId: String, documentId: String) extends Event
 final case class VerifiedAttributeUpdated(agreement: PersistentAgreement)                  extends Event
+
+// deprecated
```

Comment:
I will literally add the annotation like: 
```scala
import scala.deprecated
@deprecated("this event should not be created but just handled for retrocompatibility", "<today>")
final case class AgreementActivated(agreement: PersistentAgreement)   extends Event
```
for each of the three. If a warning appears where we handle them I will add there and just there a `@nowarn("deprecation")`

Reply:
Nice hint, but we just removed the creation of those events, so adding that annotation means putting **nowarn** everywhere.